tron/kubernetes.py

```diff
@@ -52,6 +53,23 @@ def combine_volumes(defaults: Collection[ConfigVolume], overrides: Collection[Co
     return list(result.values())
 
 
+def make_secret_volumes_serializable(secret_volumes: Collection[ConfigSecretVolume]) -> List[dict]:
```

Comment:
it'd be cool if we could make `ConfigSecretVolume` have an `_asdict` that did this for us - but reading some mypy GH issues it seems like overriding methods on namedtuples won't always typecheck/work?

this would mostly be to have this code look consistent with extra_volumes - i.e.,
```Python
...
secret_volumes=[
    volume._asdict()
    for volume in secret_volumes
],
...
```

Reply:
although, doing some testing in a python repl seems to imply that this will work!